packages/react-components/review/beta/react-components.api.md

```diff
@@ -2599,6 +2618,7 @@ export interface VideoGalleryProps {
     onUnpinParticipant?: (userId: string) => void;
     overflowGalleryPosition?: OverflowGalleryPosition;
     pinnedParticipants?: string[];
+    reactionResources?: ReactionResources;
```

Comment:
Just thinking about video tile and video gallery...
These could just be a single `reaction?: Reaction` where the interface is:
```ts
reaction: {
  name: string;
  url: string;
  frameCount: number;
  size?: number;
}
```

My thinking is that the video gallery doesn't need all the resources up front, it just needs the one reaction it is displaying.

Thoughts?

Reply:
yes. we could be passing only one type for videoTiles. Agreed.

Reply:
But for VideoGallery, we actually need them all. 
VideoGallery is accepting the resource locators once only. We get the reaction signal from `calling-stateful-components` and then we can only decide on one reaction and send to videoTile component. 
So the business logic could be shifted to videoGallery instead of VideoTile but the choice has to be made in component level.

Reply:
Great point! Yes we need them all for video gallery